index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+        <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+       <div id="main">
+         <h1 id="title">
+          Albert Einstain
+         </h1>
+       <div id="img-div"> 
```

Comment:
### Semantic tags

- It would be better to use semantic tags in this case. Ex. `<main>`